velox/exec/SharedArbitrator.cpp

```diff
@@ -370,6 +378,7 @@ uint64_t SharedArbitrator::reclaimFreeMemoryFromCandidates(
       break;
     }
   }
+  RECORD_METRIC_VALUE(kMetricArbitratorShrunkBytes, freedBytes);
```

Comment:
We probably not need kMetricArbitratorShrunkBytes and we already have reclaimed bytes stats